behat/symfony2-extension/2.1/features/bootstrap/bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require __DIR__.'/../../vendor/autoload.php';
+
+// The check is to ensure we don't use .env in production
+if (!getenv('APP_ENV')) {
```

Comment:
Maybe can you check if the Dotenv class exists? For instance in my Docker skeleton the Dotenv component isn't installed because the loading of `.env` file is handled by Docker Compose itself: https://github.com/dunglas/symfony-docker-skeleton/blob/master/composer.json